oskari-geojson/src/main/java/org/oskari/geojson/GeoJSONWriter.java

```diff
@@ -86,7 +86,8 @@ public JSONObject writeFeature(SimpleFeature f)
             }
             Object value = p.getValue();
             if (value == null) {
-                continue;
+                if (value instanceof Geometry) continue;
```

Comment:
Having `"key": null` would be better, but I'm not sure JSONObject serializes null values to anything, would have to check.